ova-compose/ova-compose.py

```diff
@@ -1191,7 +1191,7 @@ def usage():
     print("  -o, --output-file <file>    output file or directory name")
     print("  -f, --format ova|ovf|dir    output format")
     print("  -m, --manifest              create manifest file along with ovf (default true for output formats ova and dir)")
-    print("  --checksum-type sha256|sha512  set the checksum type for the manifest. Must be sha256 or sha512.")
+    print("  --checksum-type sha1|sha256|sha512  set the checksum type for the manifest. Must be sha1 or sha256 or sha512.")
```

Comment:
Just a nit - change it to "Must be sha1, sha256 or sha512."

Reply:
Done